libraries/AP_GPS/AP_GPS_UBLOX.h

```diff
@@ -71,10 +72,11 @@
 #define CONFIG_NAV_SETTINGS  (1<<10)
 #define CONFIG_GNSS          (1<<11)
 #define CONFIG_SBAS          (1<<12)
+#define CONFIG_RATE_PVT      (1<<13)
 
 #define CONFIG_ALL (CONFIG_RATE_NAV | CONFIG_RATE_POSLLH | CONFIG_RATE_STATUS | CONFIG_RATE_SOL | CONFIG_RATE_VELNED \
                     | CONFIG_RATE_DOP | CONFIG_RATE_MON_HW | CONFIG_RATE_MON_HW2 | CONFIG_RATE_RAW | CONFIG_VERSION \
-                    | CONFIG_NAV_SETTINGS | CONFIG_GNSS | CONFIG_SBAS)
+                    | CONFIG_NAV_SETTINGS | CONFIG_GNSS | CONFIG_SBAS | CONFIG_RATE_PVT)
```

Comment:
I had though of this before but dismissed it thinking I was wrong, but now I think I'm right :smile: 
If RATE_PVT is part of all configurations (and if I'm not wrong we init the unconfigured messages with CONFIG_ALL) then a GPS without support for it will never be configured (as it will never set RATE_PVT as configured).